deployability/modules/testing/tests/test_manager/conftest.py

```diff
@@ -0,0 +1,3 @@
+import pytest
+
+from ..helpers import utils
```

Comment:
Add new line at the end of the file
```suggestion
from ..helpers import utils

```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5112/commits/dd818b765d551ab2c6431656986ec60cea05225c